docs/static/transforming-data.asciidoc

```diff
@@ -450,7 +450,79 @@ For example:
     }
 --------------------------------------------------------------------------------
 
-<<plugins-filters-jdbc_streaming,jdbc_streaming>>::
+
+<<plugins-filters-jdbc_static,jdbc_static filter>>::
+
+Enriches events with data pre-loaded from a remote database.
++
+The following example fetches data from a remote database, caches it in a local
+database, and uses lookups to enhance events with data cached in the local
+database.
++
+[source,json]
+-----
+filter {
+  jdbc_static {
+    loaders => [
+      {
+        id => "remote-servers"
+        query => "select ip, descr from ref.local_ips order by ip"
+        local_table => "servers"
+      },
+      {
+        id => "remote-users"
+        query => "select firstname, lastname, userid from ref.local_users order by userid"
+        local_table => "users"
+      }
+    ]
+    local_db_objects => [
+      {
+        name => "servers"
+        index_columns => ["ip"]
+        columns => [
+          ["ip", "varchar(15)"],
+          ["descr", "varchar(255)"]
+        ]
+      },
+      {
+        name => "users"
+        index_columns => ["userid"]
+        columns => [
+          ["firstname", "varchar(255)"],
+          ["lastname", "varchar(255)"],
+          ["userid", "int"]
+        ]
+      }
+    ]
+    local_lookups => [
+      {
+        id => "local-servers"
+        query => "select descr as description from servers WHERE ip = :ip"
+        parameters => {ip => "[from_ip]"}
+        target => "server"
+      },
+      {
+        id => "local-users"
+        query => "select firstname, lastname from users WHERE userid = :id"
+        parameters => {id => "[loggedin_userid]"}
+        target => "user"
```

Comment:
also good to note that the "target" here will specify the field in which the looked up data will be stored in the event. so if there are multiple columns returning, it would be stored as a json object. the lines directly under this with the add_fields will allow you to pull fields from these json objects into top level event fields for easier analytics when it hits ES/KB.